crates/services/compression/src/storage/compressed_blocks_roots.rs

```diff
@@ -0,0 +1,57 @@
+//! The table for indexing the compressed blocks roots.
+
+use super::column::{
+    CompressionColumn,
+    MerkleizedColumnOf,
+};
+use fuel_core_storage::{
+    blueprint::plain::Plain,
+    codec::{
+        primitive::Primitive,
+        raw::Raw,
+    },
+    merkle::sparse::MerkleizedTableColumn,
+    structured_storage::TableWithBlueprint,
+    Mappable,
+    MerkleRoot,
+};
+
+use fuel_core_types::fuel_types::BlockHeight;
+
+/// The table for the compressed blocks sent to DA.
+pub struct CompressedBlocksRoots;
```

Comment:
Is this table used? Since we use the `Merkleized` table wrapper we should not need a dedicated table for the roots, as they'll be in the `MerkleMetadata<...>` table for the associated table column.

Reply:
(This may be different if we chose to go with BMTs instead of SMTs)